2024-11-21 13:10:38,018 [DEBUG] DroneApp has started! Directory ./
2024-11-21 13:10:38,912 [ERROR] 'simulator_ip'
Traceback (most recent call last):
  File "/home/gourav/raspi-app/app.py", line 74, in <module>
    drone = Drone(config)
  File "/home/gourav/raspi-app/drone.py", line 11, in __init__
    simulator_ip = configurations['drone']['simulator_ip']
  File "/usr/lib/python3.9/configparser.py", line 1254, in __getitem__
    raise KeyError(key)
KeyError: 'simulator_ip'
2024-11-21 13:10:40,918 [ERROR] 'simulator_ip'
Traceback (most recent call last):
  File "/home/gourav/raspi-app/app.py", line 74, in <module>
    drone = Drone(config)
  File "/home/gourav/raspi-app/drone.py", line 11, in __init__
    simulator_ip = configurations['drone']['simulator_ip']
  File "/usr/lib/python3.9/configparser.py", line 1254, in __getitem__
    raise KeyError(key)
KeyError: 'simulator_ip'
2024-11-21 13:10:42,922 [ERROR] 'simulator_ip'
Traceback (most recent call last):
  File "/home/gourav/raspi-app/app.py", line 74, in <module>
    drone = Drone(config)
  File "/home/gourav/raspi-app/drone.py", line 11, in __init__
    simulator_ip = configurations['drone']['simulator_ip']
  File "/usr/lib/python3.9/configparser.py", line 1254, in __getitem__
    raise KeyError(key)
KeyError: 'simulator_ip'
2024-11-21 13:10:44,926 [ERROR] 'simulator_ip'
Traceback (most recent call last):
  File "/home/gourav/raspi-app/app.py", line 74, in <module>
    drone = Drone(config)
  File "/home/gourav/raspi-app/drone.py", line 11, in __init__
    simulator_ip = configurations['drone']['simulator_ip']
  File "/usr/lib/python3.9/configparser.py", line 1254, in __getitem__
    raise KeyError(key)
KeyError: 'simulator_ip'
